packages/components/src/tile/src/Tile.css

```diff
@@ -1,11 +1,11 @@
 .o-ui-tile {
     width: 100%;
     min-width: 100px;
-    border: 1px solid var(--o-ui-b-alias-low-break);
-    background-color: var(--o-ui-bg-alias-default);
+    border: none;
```

Comment:
On peut surement juste enlever cette ligne la?

Reply:
Buttons have a border by default. This removes it.